app/src/main/java/com/example/vybe/MapFragment.java

```diff
@@ -58,8 +54,25 @@
     private MapView mMapView;
     private FirebaseFirestore db = FirebaseFirestore.getInstance();
     private GoogleMap mMap;
-    private VibeEvent vibe;
+    private OnMapFragmentReadyListener onMapFragmentReadyListener;
+
+//    private LocationController locationController;
+
```

Comment:
why is this commented out?

Reply:
The `getUserLocation()` method in `LocationController` is now static so we no longer need instances of LocationController

Reply:
I'll get rid of the comment tho